.github/CODEOWNERS

```diff
@@ -1,100 +1,100 @@
 docs/ @vectordotdev/ux-team
```

Comment:
Do we still have a `ux-team`?

Reply:
Nominally, yes. I'd like to keep this one to have a smaller group continue to build up sufficient context to steer Vector's UX.